mkdocs.yml

```diff
@@ -36,6 +36,7 @@ nav:
     - Subsystems: plugin/subsystems.md
     - ProxyPool: plugin/proxypool.md
     - Private COM-RPC : plugin/private-comrpc.md
+    - Split Implementation: plugin/split-implementation.md
```

Comment:
Wanted to read this but seems it is till missing. In the preview I got a deadlink..

Reply:
That's because I haven't written that page yet :) 

Plan to base it off this existing page in rdkcentral: https://wiki.rdkcentral.com/pages/viewpage.action?pageId=198262351.